TODO.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Hugo Starter Checklist
+
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md):
+
+- [ ] 1. Website should be hosted in an open source repo
+  - [ ] Hosted in the same organization as the main project
+- [ ] 2. Reference the origin company correctly
```

Comment:
Is this a todo? Or a "do if you need to?"

Reply:
Some of these are optional depending on situation (some projects, I assume, don't have an origin company)